Sources/DataPipeline/CustomerIO.swift

```diff
@@ -0,0 +1,31 @@
+import CioInternalCommon
+
+public extension CustomerIO {
+    /**
+     Initialize the shared `instance` of `CustomerIO`.
+     Call this function when your app launches, before using `CustomerIO.instance`.
+     */
+    @available(iOSApplicationExtension, unavailable)
+    static func initialize(
+        writeKey: String,
+        configure configureHandler: ((inout SdkConfig, inout DataPipelineConfigOptions) -> Void)?
+    ) {
+        var sdkConfig = SdkConfig.Factory.create(siteId: "", apiKey: "", region: .US)
```

Comment:
we shouldnt be needing `sdkConfig` in the datapipeline?

Reply:
Yeah, added it for `DIGraph`. But will keep looking to remove the need of it in `DataPipeline`.